setup.cfg

```diff
@@ -66,13 +66,16 @@ testing =
 	filelock>=3.4.0
 	ini2toml[lite]>=0.9
 	tomli-w>=1.0.0
+	tomli
```

Comment:
I suppose that this is to cover https://github.com/pypa/setuptools/blob/0435c72fe989230f2549b15b0fc2b0a52f776e8f/setuptools/tests/integration/test_pip_install_sdist.py#L203?

If we are going for that approach, would it make more sense to remove the [`testing-integration`](https://github.com/pypa/setuptools/blob/0435c72fe989230f2549b15b0fc2b0a52f776e8f/setup.cfg#L77) extra and just use `testing` for everything?

Reply:
I don't see an issue with having less dependencies in `testing-integration`. And if you wanna merge them together that's up to you.

If you want to reduce duplication, then I think my latest commit should help with that (assuming it works)

Reply:
Hi @Avasam, I was just asking your option about the matter, sorry if it prompted a change.

I think what we do in the PR is to just leave as you wrote it before: not touching `testing-integration` and just adding the extras in `testing`.

In a separated PR, I think we just remove the `testing-integration` and use `testing` everywhere...

Although it might add a bit of overhead for the integration tests, it will simplify and streamline the setup... This way we reduce the divergence with `skeleton`. The additional overhead in the integration tests should not be too much, and the integration tests just run before the releases anyway.

Reply:
> sorry if it prompted a change.

I actually have the deduplication commit already ready, so it's more of a coincidence ^^

> I think what we do in the PR is to just leave as you wrote it before

np, I'll revert that change and leave modifications to `setup.cfg` that are not absolutely necessary to another PR.

> The additional overhead in the integration tests should not be too much

Especially if the pypi downloads are cached anyway (are they?)

Reply:
> Especially if the pypi downloads are cached anyway (are they?)

I don't remember now if the CI is keeping the pip cache 😅. But it should not matter that much.

I just hope someone is not installing `setuptools[testing-integration]`. But that is unlikely in my opinion.